src/components/Navigationbar/Navigationbar.tsx

```diff
@@ -0,0 +1,101 @@
+import image from './img/logo.png'
+import backgroundImage from '@/components/Navigationbar/img/Rectangle 49.png'
+import Image from 'next/image';
+import Link from 'next/link';
+import styles from './styles.module.scss';
+import { url } from 'inspector';
+const Navigationbar = () => {
+
+    return (
+        <div className={styles.container}
+    
+        >
+            {/* <Image src={img} alt="imagen de cabecera" /> */}
+
+            <div className={styles.nav}>
+                <Image src={image} alt="imagen de footer" width={213} height={74} className={styles.image}/>
+                <nav>
+                    <ul className={styles.nave}>
+                        <li className={styles.naves}>
+                            <svg xmlns="http://www.w3.org/2000/svg" width="50" height="39" viewBox="0 0 50 39" fill="none">
+                                <path d="M49.5328 38.6679H0V3.07975H3.45321C3.45321 2.22804 3.48219 1.44005 3.44308 0.654969C3.41991 0.176964 3.56765 0.0205265 4.05144 0.00893854C9.12119 -0.118529 13.8462 1.11994 18.2714 3.55631C20.5383 4.80491 22.5632 6.37653 24.3681 8.23206C24.4883 8.35518 24.5738 8.51452 24.7809 8.80422C24.9605 8.529 25.0358 8.36242 25.1561 8.24075C29.9433 3.39987 35.7649 0.749121 42.5163 0.101642C43.5288 0.00459295 44.5558 0.0509449 45.5755 0.0176294C45.955 0.00459297 46.1028 0.134958 46.0883 0.536192C46.0593 1.35894 46.0796 2.18313 46.0796 3.05078H49.5357V38.6679H49.5328ZM25.6457 35.7651C31.2108 31.521 37.5509 29.9552 44.2661 29.5931V1.72831C43.2956 1.82101 42.3642 1.8746 41.4459 2.00497C35.394 2.86103 30.2229 5.4437 26.0092 9.89928C25.7485 10.1745 25.6384 10.4512 25.6384 10.835C25.6486 18.9365 25.6457 27.0365 25.6457 35.1379V35.7666V35.7651ZM5.26239 1.68195V29.5974C12.0196 29.9523 18.3452 31.5413 23.8857 35.7579C23.8857 35.5276 23.8857 35.4088 23.8857 35.2886C23.8857 27.1002 23.8901 18.9118 23.8698 10.7235C23.8698 10.4483 23.7003 10.1166 23.5062 9.90797C19.6735 5.7812 14.9384 3.28399 9.43262 2.23238C8.07537 1.9731 6.68772 1.86591 5.26239 1.68051V1.68195ZM47.7642 36.8833V4.88169H46.0782C46.0782 5.14241 46.0782 5.36838 46.0782 5.5929C46.0782 13.9725 46.0738 22.352 46.0883 30.7316C46.0883 31.2009 45.9739 31.337 45.4843 31.3718C43.4332 31.5167 41.3676 31.6123 39.3397 31.9309C35.155 32.5871 31.2398 34.0081 27.7387 36.4531C27.5765 36.5676 27.4215 36.6921 27.1709 36.8833H47.7642ZM22.3286 36.8776C22.1026 36.7008 21.9679 36.5676 21.81 36.4763C20.5788 35.7608 19.3852 34.967 18.1033 34.3586C13.684 32.2655 8.97779 31.4312 4.11808 31.3689C3.60096 31.3617 3.44018 31.2443 3.44163 30.6954C3.46191 22.3129 3.45466 13.9305 3.45466 5.54799C3.45466 5.32782 3.45466 5.1062 3.45466 4.88748H1.77006V36.8776H22.3286Z" fill="white"/>
```

Comment:
this part of your code is so bad, that you need to use the icons directly exporting from the SVG 
```
import icon from 'path/of/the/icon.svg'
```
to use 
```
<icon />
```